owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -58,13 +60,16 @@ public class WhatsNewActivity extends FragmentActivity implements ViewPager.OnPa
     private ProgressIndicator mProgress;
     private ViewPager mPager;
 
+    private WhatsNewActivityBinding binding_activity;
+    private static WhatsNewElementBinding binding_element;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.whats_new_activity);
```

Comment:
Now that we are using ViewBinding, we must tell the `setContentView` method:
```suggestion
        setContentView(binding_activity.getRoot());
```